File: src/main/java/com/monsource/bis/account/model/Account.java
package com.monsource.bis.account.model;

import com.monsource.bis.data.entity.type.AccountStatus;
import com.monsource.bis.data.entity.type.Gender;

import java.util.List;

public class Account {

	private Integer id;
	private String username;
	private String name;
	private Gender gender;
	private String email;
	private String phone;
	private Integer cityId;
	private Integer districtId;
	private String address;
	private AccountStatus status;
	private List<Group> groups;

	public Integer getId() {
		return this.id;
	}

	public void setId(Integer id) {
		this.id = id;
	}

	public String getUsername() {
		return this.username;
	}

	public void setUsername(String username) {
		this.username = username;
	}

	public String getName() {
		return this.name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public Gender getGender() {
		return this.gender;
	}

	public void setGender(Gender gender) {
		this.gender = gender;
	}

	public String getEmail() {
		return this.email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	public String getPhone() {
		return this.phone;
	}

	public void setPhone(String phone) {
		this.phone = phone;
	}

	public Integer getCityId() {
		return this.cityId;
	}

	public void setCityId(Integer cityId) {
		this.cityId = cityId;
	}

	public Integer getDistrictId() {
		return this.districtId;
	}

	public void setDistrictId(Integer districtId) {
		this.districtId = districtId;
	}

	public String getAddress() {
		return this.address;
	}

	public void setAddress(String address) {
		this.address = address;
	}

	public AccountStatus getStatus() {
		return this.status;
	}

	public void setStatus(AccountStatus status) {
		this.status = status;
	}

	public List<Group> getGroups() {
		return this.groups;
	}

	public void setGroups(List<Group> groups) {
		this.groups = groups;
	}

}

File: src/main/webapp/app/account/model/Account.js
Ext.define('Account.model.Account', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'id', type: 'int'},
        {name: 'username', type: 'string'},
        {name: 'password', type: 'string'},
        {name: 'name', type: 'string'},
        {name: 'gender', type: 'string'},
        {name: 'email', type: 'string'},
        {name: 'phone', type: 'string'},
        {name: 'status', type: 'string'},
        {name: 'address', type: 'string'},
        {name: 'cityId', type: 'int'},
        {name: 'districtId', type: 'int'}
    ],
    hasMany: {model: 'Account.model.Group', name: 'groups',foreignKey: 'accountId',
        primaryKey : 'id'},
    proxy: {
        type: 'ajax',
        url: '/account-mod/account/single.json',
        reader: {
            type: 'json',
            root: 'data'
        }
    }
})